flink-table/flink-table-planner-blink/src/main/java/org/apache/calcite/sql2rel/AuxiliaryConverter.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to you under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.calcite.sql2rel;
+
+/*
+ * THIS FILE HAS BEEN COPIED FROM THE APACHE CALCITE PROJECT UNTIL CALCITE-1761 IS FIXED.
```

Comment:
CALCITE-1761 is already fixed

Reply:
I think it should be the following reasons:
1.We need support extended TUMBLE_ROWTIME and so on in FlinkSqlOperatorTable.
2.To change to the implementation of calcite, we need to modify a lot of things (and in operators, different ways of handling window properties), and possibly SESSION_END may be defined differently.

I'll add reason one to comment.